client/src/component/BlogCard/style.js

```diff
@@ -5,12 +5,23 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     justifyContent: 'center',
     flexDirection: 'row',
-    marginTop: '2rem',
+    marginTop: '3rem',
     width: '75%',
+    [theme.breakpoints.down('md')]: {
+      flexDirection: 'column',
+    },
+  },
+  left: {
+    width: '100%',
+    marginRight: '2rem',
   },
-
   image: {
-    width: '10em',
+    [theme.breakpoints.down('sm')]: {
+      width: '250px',
+      height: 'auto',
+    },
+
+    width: '300px',
```

Comment:
change it to 250px because we have images with width resolution below 300px, and doesn't render properly